manifests/init.pp

```diff
@@ -13,8 +13,20 @@
 #
 # $group::                         Override the name of the puppet group.
 #
+# $dir_prefix::                    Override the where to find the data directory on Windows.
```

Comment:
Yeah, I thought that was the direction you were going in.  This works too, it's just a little more work.